### PR TITLE
OSD-23975 add version and state in the health-check metrics 

### PR DESCRIPTION
**Pull request by tkong-redhat**
Add version and state in the health-check metrics so the health_check could be exposed to RHOBS

### CODE REVIEW

**Comment by ravitri** [comment]
/hold

@tkong-redhat - Can we include the changes related to https://github.com/openshift/managed-upgrade-operator/pull/445 as well? The PR should merge in a while

File: pkg/upgraders/healthcheckstep.go [hunk 1]
```diff
@@ -119,5 +122,28 @@ func (c *clusterUpgrader) PostUpgradeHealthCheck(ctx context.Context, logger log
 	if err != nil || !ok {
 		return false, err
 	}
+
 	return true, nil
 }
+
+func getCurrentVersion(ug *upgradev1alpha1.UpgradeConfig) string {
```
Comment (ravitri) [inline pkg/upgraders/healthcheckstep.go:129]: Can we use ClusterVersion version here?

Comment (tkong-redhat) [reply pkg/upgraders/healthcheckstep.go:129]: Nice suggestion. Let me have a look

**Comment by ravitri** [comment]
/lgtm
/approve

Thanks a lot Tony!

**Pull request merged**